### ISSUE TITLE
Definitions' scopes are too broad

### ISSUE THREAD

**Issue by waldoj** [openva/va-decoded#25]
A lot of definitions are scoped globally, despite that there's nothing in the text that should lead to that. My best guess is that this is from `State::extract_definitions` opening with `$scope = 'global'`, setting the default scope to global. (It's also possible that we start with this for perfectly good reasons.) Step through the code while scraping definitions to figure out what's up.

**Comment by waldoj** [comment]
Well, that didn't work. We we have no definitions appearing at all.

**Comment by waldoj** [comment]
Ah, I think the problem is from #26. Perhaps this fix _did_ work.

**Comment by waldoj** [comment]
This looks a bit better

```
SELECT scope, COUNT(*) AS number FROM dictionary GROUP BY scope;
+---------+--------+
| scope   | number |
+---------+--------+
| Section |    148 |
| article |    260 |
| chapter |    547 |
| part    |    108 |
| section |    997 |
| title   |     26 |
+---------+--------+
```

I mean, other than the obvious case problem (`Section` vs. `section`) and the complete lack of any global definitions. Still, the presence of `article` and `part` both look great. The low numbers are also because of #26.

**Comment by waldoj** [comment]
This looks better:

```
SELECT scope, COUNT(*) AS number FROM dictionary GROUP BY scope;
+----------+--------+
| scope    | number |
+----------+--------+
| article  |   1474 |
| chapter  |   4402 |
| part     |    143 |
| section  |   1949 |
| subtitle |     21 |
| title    |    473 |
+----------+--------+
```

Except that there are still no global definitions.

**Comment by waldoj** [comment]
Ah-ha—the problem was `GLOBAL_DEFINITIONS` in `config.inc.php`. I had this:

``` php
define('GLOBAL_DEFINITIONS', '1,2.1');
```

instead of this:

``` php
define('GLOBAL_DEFINITIONS', '1,2.1,2');
```

Previously, we used the SGML, which only included titles and chapters. But now the XML includes subtitles, parts, and articles, and definitions are found within Article 2 of Title 1, Chapter 2.1. So the prior scope wasn't working. I updated `GLOBAL_DEFINITIONS`, started a re-parse, and global definitions are streaming in now.
